magento/magento2
# Fixed Cache Management mass-action checkbox alignment

### Description (*)

Fixed Cache Management mass-action checkbox alignment
### Fixed Issues (if relevant)

1. magento/magento2#21255: Magento backend Cache Management mass action checkbox alignment is not proper.

### Manual testing scenarios (*)

1. Login to admin panel
2. Navigate on System -> Tools (Cache Management)
3. Observe the mass action check-box

### Contribution checklist (*)
